checkout/access-checkout-android/src/test/java/com/worldpay/access/checkout/AccessCheckoutCardTest.kt

```diff
@@ -402,7 +402,7 @@ class AccessCheckoutCardTest {
     @Test
     fun givenCVVHasBeenUpdatedToPartiallyValidThenCardListenerIsNotifiedOfResult() {
         val updatedCVV = "1234"
-        val cardBrand = CardBrand("test", emptyList(), null, emptyList())
+        val cardBrand = CardBrand(name = "test", images = emptyList())
```

Comment:
images is already an empty list by default

Reply:
updated